src/frontend/data/user-cases.tsx

```diff
@@ -0,0 +1,13 @@
+import { checkBalanceStepList, contributeStepList} from './user-cases/';
```

Comment:
I think that this is probably redundant. We only have 2 use cases (not user-cases like the filename), and even in the future may only add 1 more. So I think the enum in state.ts is probably enough to cover all the bases.

Reply:
Sounds good, will remove in next commit.